keywords/RemoteExecutor.py

```diff
@@ -51,13 +54,14 @@ def execute(self, commamd):
         log_info("Closing connection to {}".format(self.host))
         self.client.close()
 
-        return status
+        return status, stdout_p, stderr_p
 
-    def must_execute(self, command):
+    def must_execute(self, command, any_status=False):
```

Comment:
For applications where any_status is allowed, could you just use `self.execute`? The whole idea of `must_execute` is that any status failure will cause an exception to be thrown.